src/certificates/services/certificates.service.ts

```diff
@@ -18,12 +26,47 @@ export class CertificatesService {
     });
   }
 
-  async create(createCertificateDto: CreateCertificateDto) {
+  async getAllByUserId(
+    query: SearchCertificatesQueryDto,
+  ): Promise<CertificateResponseDto[]> {
+    const { userId } = query;
+
+    const certificates = await this.prismaService.certificate.findMany({
+      where: {
+        userAttribute: {
+          User: {
+            id: userId,
+          },
+        },
+      },
+    });
+
+    return certificates.map((certificate) => ({
+      id: certificate.id,
+      key: certificate.key,
+      type: certificate.type,
+      validTill: certificate.validTill,
+    }));
+  }
+
+  async create(userId: number, createCertificateDto: CreateCertificateDto) {
     const user = await this.prismaService.user.findUnique({
-      where: { id: createCertificateDto.userId },
+      where: { id: userId },
     });
```

Comment:
I assume we are not reading user_id from JWT because admin can create certificate for some other user?